### DISCUSSION TITLE
[Icon Request] Handshake

### DISCUSSION THREAD

**Discussion by viniciussbs** [tailwindlabs/heroicons#428] [Ideas]
Great for content about volunteering.

**Comment by simonswiss** [comment]
Need a 2020-friendly "elbow bump" variant too I guess 😅

**Comment by viniciussbs** [comment]
You're right, @simonswiss. Maybe something like these from Font Awesome:

https://fontawesome.com/icons/hands-heart?style=duotone
https://fontawesome.com/icons/hand-holding-heart?style=solid

They agre great for volunteering too. I could change the title of the issue to "Volunteering".
